tests/system_tests_autolinks.py

```diff
@@ -254,15 +255,18 @@ def schedule_auto_link_reconnect_test(self):
                 Timer(self.timer_delay, self.check_auto_link).start()
 
     def test_auto_link_reattach(self):
-        long_type = 'io.skupper.router.router.config.autoLink'
-        query_command = 'QUERY --type=' + long_type
-        output = json.loads(self.run_skmanage(query_command))
-
-        # Since the distribution of the autoLinked address 'examples'
-        # is set to unavailable, the link route will initially be in the
-        # failed state
-        self.assertEqual(output[0]['operStatus'], 'failed')
-        self.assertEqual(output[0]['lastError'], 'Node not found')
+        def check_autolink_status():
+            long_type = 'io.skupper.router.router.config.autoLink'
+            query_command = 'QUERY --type=' + long_type
+            output = json.loads(self.run_skmanage(query_command))
+
+            # Since the distribution of the autoLinked address 'examples'
+            # is set to unavailable, the link route will initially be in the
+            # failed state
```

Comment:
According to the failure message, the link route is initially in the `attaching` state, and then it later stabilizes as `failed`. (I understand that in this comment, "initially" is supposed to mean "before we run the next part of the test", but still)